tests/test_hf_api.py

```diff
@@ -928,6 +929,31 @@ def test_model_info_with_security(self):
             {"containsInfected": False},
         )
 
+    @with_production_testing
+    def test_model_info_with_file_metadata(self):
```

Comment:
I would also add another test that shows that the metadata is not included in the default call

Reply:
(this is testing the server more than the client though)